websites/P/Phind/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+		clientId: "1106990410838065172",
+	}),
+	slideshow = presence.createSlideshow(),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const assets = {
+			phindLogo: "https://i.imgur.com/fakWcYA.png",
+			search: "https://i.imgur.com/a5qsEbL.png",
+		},
+		presenceData: PresenceData = {
+			largeImageKey: assets.phindLogo,
+			details: "Browsing Phind",
+		},
+		pathDetailsMap = {
+			"/default": "Viewing how to set Phind as default",
+			"/bangs": "Viewing !Bangs",
+			"/mobile": "Viewing Mobile Page",
+			"/hotkeys": "Viewing Hotkeys",
+			"/history": "Viewing Search History",
+			"/about": "Viewing About Page",
+			"/tutorial": "Viewing Tutorial",
+			"/privacy": "Viewing Privacy Policy",
+			"/terms": "Viewing Terms",
+		},
+		{ pathname, href } = document.location,
+		pathDetails = Object.entries(pathDetailsMap).find(([pathPrefix]) =>
+			pathname.startsWith(pathPrefix)
+		),
+		searchResults = document.querySelectorAll("div.row[name^='answer-']"),
+		[displayTime, displaySearch, cycleSearch, shareSearch, privateMode] =
+			await Promise.all([
+				presence.getSetting("displayTime"),
+				presence.getSetting("displaySearch"),
+				presence.getSetting("cycleSearch"),
+				presence.getSetting("shareSearch"),
+				presence.getSetting("privateMode"),
+			]);
+
+	if (displayTime) presenceData.startTimestamp = browsingTimestamp;
+
+	if (pathDetails) {
+		presenceData.details = pathDetails[1]; // display text of /path
+		presenceData.state = pathname; // display /path
+		slideshow.deleteAllSlides();
+	} else if (searchResults.length > 0 && !privateMode) {
+		// we must be on /search
+		presenceData.details = displaySearch ? "Searching for:" : "Searching";
+		presenceData.smallImageKey = assets.search;
```

Comment:
Use standardized asset
```suggestion
		presenceData.smallImageKey = Assets.Search;
```